runtime/codert_vm/decomp.cpp

```diff
@@ -1765,6 +1765,20 @@ UDATA
 osrFrameSize(J9Method *method)
 {
 	J9ROMMethod *romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(method);
+	return osrFrameSizeRomMethod(romMethod);
+}
```

Comment:
To minimize discrepancies between jitaas and mainline this code needs to be delivered to the master branch.